run/helm/snap/templates/daemonset.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: snap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  template:
+    metadata:
+      name: snap
+      labels:
+        daemon: snapteld
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repo }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        securityContext:
+          privileged: {{ .Values.privileged }}
+        ports:
+        - containerPort: 8181
+        livenessProbe:
+          httpGet:
+            path: /v1/plugins
+            port: 8181
+          initialDelaySeconds: 60
+          timeoutSeconds: 5          
+        readinessProbe:
+          httpGet:
+            path: /v1/plugins
+            port: 8181
+          initialDelaySeconds: 60
+          timeoutSeconds: 5          
+        env:
+          - name: INFLUXDB_USER
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Chart.Name }}-influxdb-auth
+                key: influxdb-user
+          - name: INFLUXDB_PASS
```

Comment:
Yes, it is handled here https://github.com/intelsdi-x/snap-integration-kubernetes/blob/master/src/helm/start.sh#L18.